src/test/java/api/support/fixtures/CheckOutFixture.java

```diff
@@ -0,0 +1,127 @@
+package api.support.fixtures;
+
+import static api.support.APITestContext.getOkapiHeadersFromContext;
+import static api.support.http.InterfaceUrls.checkOutByBarcodeUrl;
+import static api.support.http.InterfaceUrls.overrideCheckOutByBarcodeUrl;
+
+import java.util.UUID;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.folio.circulation.support.http.client.Response;
+import org.joda.time.DateTime;
+
+import api.support.RestAssuredClient;
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import io.vertx.core.json.JsonObject;
+
+import api.support.builders.OverrideCheckOutByBarcodeRequestBuilder;
+
+public class CheckOutFixture {
+  private final UsersFixture usersFixture;
+  private final ServicePointsFixture servicePointsFixture;
+  private final RestAssuredClient restAssuredClient;
+
+  public CheckOutFixture(
```

Comment:
Given that this is considered a new file by git, please could the parameter formatting be fixed in this initial version

(The incorrect formatting is likely something I did, long ago, apologies :-/ )